packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -703,6 +730,61 @@ - (NSDictionary *)encodingMap
 
 #pragma mark - Internal methods
 
+- (EXFileSystemUploadType)_importUploadType:(NSNumber * _Nullable)type
+{
+  switch ([type intValue]) {
+    case EXFileSystemBinaryContent:
+      return EXFileSystemBinaryContent;
+    case EXFileSystemMultipart:
+      return EXFileSystemMultipart;
+  }
+  
+  return EXFileSystemInvalidType;
```

Comment:
That looks terrible TBH ... 😉 
I've googled a bit and found out some other tricky and not-so-pretty way:
```suggestion
  switch ([type intValue]) {
    case EXFileSystemBinaryContent:
    case EXFileSystemMultipart:
      return [type intValue];
  }
  
  return EXFileSystemInvalidType;
```